aten/src/ATen/native/cpu/Depthwise.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <ATen/Tensor.h>
+
+/*
+  Caffe 2 depthwise 3x3 convolution operator
+*/
+
+namespace at {
+namespace native {
+
+Tensor depthwise3x3ConvNCHW(
```

Comment:
looks like the convention under aten/src is that all numeric functions are registered in "src/ATen/native/native_functions.yaml" and codegen will produce function declarations at one centralized place.

lowercase and prefixed "_" for internal functions (no python binding). so this probably should be something like "_convolution_depthwise3x3_winograd"

These are my understanding though - @ezyang or @gchanan might give more accurate review.

Reply:
That's correct!